media/js/base/site.js

```diff
@@ -192,6 +197,13 @@
             h.className += ' is-firefox';
         }
 
+        // Add class to reflect browsers that get 1st class JS & CSS support.
+        var isModernBrowser = window.site.isModernBrowser = window.site.cutsTheMustard();
+
+        if (isModernBrowser) {
+            h.className += ' is-modern-browser';
```

Comment:
You can use `classList.add('is-modern-browser');` if it cuts the mustard!

Reply:
(not a blocker)

Reply:
I'm going to try and resist doing this. Even though it's fine to use `classList` in this one place, unless someone follows the code carefully it could be easy to assume the API is ok to use in the rest of this file.